packaging/common/BUILD

```diff
@@ -0,0 +1,73 @@
+nfpm_content_dirs(
+    name="root_dirs",
+    dirs=[
+        # "/etc/bash_completion.d",
+        # "/etc/logrotate.d",
+        "/etc/st2",
+        "/opt/stackstorm",
+        "/opt/stackstorm/st2",
+        "/usr/share/doc/st2",
+    ],
+    file_owner="root",
+    file_group="root",
+    file_mode="rwxr-xr-x",
+)
+
+nfpm_content_dirs(
+    name="packs_group_dirs",
+    dirs=[
+        "/opt/stackstorm/packs",
+        "/opt/stackstorm/virtualenvs",
+        "/usr/share/doc/st2/examples",
+    ],
+    file_owner="root",
+    file_group=ST2_PACKS_GROUP,
+    file_mode="rwxrwxr-x",
+)
+
+nfpm_content_dirs(
+    name="st2_user_dirs",
+    dirs=[
+        "/opt/stackstorm/configs",
+        "/opt/stackstorm/exports",
+        "/opt/stackstorm/overrides",
+        "/var/log/st2",
+        "/var/run/st2",
+    ],
+    file_owner=ST2_SVC_USER,
+    file_group="root",
+    file_mode="rwxr-xr-x",
+)
+
+target(
+    name="dirs",
+    dependencies=[
+        ":root_dirs",
+        ":packs_group_dirs",
+        ":st2_user_dirs",
+    ],
+)
```

Comment:
Each set of directories above (`:root_dirs`, `:packs_group_dirs`, `:st2_user_dirs`) defines the relevant permissions. They are separate targets because those permissions differ. Then, this target makes it easy to depend on all of the directories in the `nfpm_*_packages` targets.